### ISSUE TITLE
question about labels of inference (predict) data and thread

### ISSUE THREAD

**Issue by liujing0055** [automl/auto-sklearn#498]
Hi，Auto-sklearm Team:
After scanning the code， I found that the data were divided into two parts,training(fit) and inference(predict) when calling the fit function as shown below.
**automl.fit(data, label,feat_type=feature_types)** 
 But I could not find the variable of label of inference(predict),Could you please tell me where I can find the variable?
Another question is about thread, is there any possible way to run the program with only one thread?
Thanks.

**Comment by liujing0055** [comment]
I need to cheek the difference between my own designed algorithm and sklearn's algorithm.Could you please tell me the reason why not easy?

**Comment by mfeurer** [comment]
Because there is no interface to access the data.

**Comment by mfeurer** [comment]
Good news, this is actually possible using scikit-learn's internal splitter and we finally have an example for doing so: https://automl.github.io/auto-sklearn/development/examples/40_advanced/example_resampling.html#scikit-learn-splitter-objects

The example is currently only available on the documentation of the development branch, but the example also works on the master branch.
